concordium-consensus/src/Concordium/KonsensusV1/Consensus/Timeout.hs

```diff
@@ -319,7 +315,7 @@ executeTimeoutMessage (PartiallyVerifiedTimeoutMessage{..})
                 then -- Advance the round with the new certified block.
                     advanceRoundWithQuorum newCertifiedBlock
                 else -- Otherwise we just update the highest certified block.
```

Comment:
I suppose the comment isn't exactly accurate anymore.

Reply:
Fixed.